docs/REST API/rest-api.md

```diff
@@ -5934,7 +5934,14 @@ load balancer timeout.
         }
       ]
     }
-  ]
+  ],
+   "stats": {
+     "system_time_p50": 1.32,
+     "system_time_p95": 4.02,
+     "user_time_p50": 3.55,
+     "user_time_p95": 3.00,
+     "total_executions": 3920
+  }
```

Comment:
Live queries have two APIs, one synchronous and one asynchronous.

The asynchronous one (in the "API for contributors") should be the one being amended for #467. Though the goal for #467 has since been reduced to only collect stats for saved queries when ran as live queries (no API changes), so maybe this PR is not valid anymore.